test/remote/gateways/remote_qvalent_test.rb

```diff
@@ -0,0 +1,106 @@
+require "test_helper"
+
+class RemoteQvalentTest < Test::Unit::TestCase
+  def setup
+    Base.mode = :test
```

Comment:
FYI, this is done by `test_helper.rb`, so not necessary here. I'll remove when I merge.